encrypted-media-respec.html

```diff
@@ -2992,8 +2992,9 @@ <h2>
               <p>
                 The HDCP Policy is represented by {{MediaKeysPolicy/minHdcpVersion}}. If the system
                 can enable the HDCP version specified or higher, then the policy will result in a
-                {{MediaKeyStatus}} of {{MediaKeyStatus/"usable"}}. Implementations MUST support
-                {{MediaKeysPolicy/minHdcpVersion}} values defined in [[EME-HDCP-VERSION-REGISTRY]].
+                {{MediaKeyStatus}} of {{MediaKeyStatus/"usable"}}. The [[EME-HDCP-VERSION-REGISTRY]]
```

Comment:
This would add the registry to the list of normative references because the paragraph appears in a normative section of the spec (see [ReSpec doc](https://respec.org/docs/#normative-vs-informative-references)). The registry being a non-normative document, it should rather appear in the list of informative references.

```suggestion
                {{MediaKeyStatus}} of {{MediaKeyStatus/"usable"}}. The [[?EME-HDCP-VERSION-REGISTRY]]
```

(The [published Recommendation](https://www.w3.org/TR/encrypted-media/) does that correctly for the registries it references, even though the spec incorrectly uses normative statements for them as you noted in #535).